src/atoms/BookingConfirmation/BookingConfirmation.tsx

```diff
@@ -1,8 +1,8 @@
 import { FC } from 'react';
```

Comment:
@bejoinka @sharlotta93 Note: this component/stage is not used anymore in the Scheduling flow as showing confirmation of the booking happens now in a Message action. So in theory this component could also be removed.